guides/common/modules/proc_creating-hosts-with-uefi-http-boot-provisioning.adoc

```diff
@@ -3,70 +3,48 @@
 [id="creating-hosts-with-uefi-http-boot-provisioning"]
 = Creating hosts with UEFI HTTP boot provisioning
 
-You can provision hosts from {Project} using the UEFI HTTP Boot.
+You can provision hosts from {Project} by using the UEFI HTTP Boot if the hosts have this capability.
+In HTTP boot, configuration files are transferred over HTTP instead of TFTP as in PXE boot.
+Using this method can help reduce the booting time during host provisioning.
```

Comment:
In addition to being faster, we've also had reports that for large files (like install images) it's also more reliable than TFTP.